Sources/BraintreeVenmo/BTVenmoClient.swift

```diff
@@ -220,7 +228,7 @@ import BraintreeCore
 
     /// Returns true if the proper Venmo app is installed and configured correctly, returns false otherwise.
     @objc public func isVenmoAppInstalled() -> Bool {
-        guard let appSwitchURL = BTVenmoAppSwitchRedirectURL().baseAppSwitchURL else {
+        guard let appSwitchURL = BTVenmoAppSwitchRedirectURL.baseAppSwitchURL else {
```

Comment:
random: curious if we could leverage [#URL macros](https://www.avanderlee.com/swift/macros/#what-are-swift-macros) in iOS? It's new I'm not sure if there are minimum iOS version requirements, but it'd be cool if we could eliminate `if let` for the static URLs.